tee attributes.md

```diff
@@ -0,0 +1,16 @@
+## TEE Attributes
+
+Required properties to be called a “TEE”:
+* ___Data integrity___ - data values stored or being operated upon within the bounds of the TEE cannot be modified by an actor operating outside the TCB.
+* ___Data confidentiality___ - data values stored or being operated upon within the bounds of the TEE cannot be determined by an actor operating outside the TCB.
+* ___Code integrity___ - software codes executed inside the bounds of the TEE cannot be modified by an actor operating outside the TCB.
+
+TEE's may have other properties:
+* _Code confidentiality_ - software codes executed inside the bounds of the TEE cannot be modified by an actor operating outside the TCB.
+* _Programmability_ - the extent to which the software codes operated upon inside the bounds of the TEE can be provided by an identified actor.
```

Comment:
"can be provided" I think should be explicitly "after time of manufacture" or something similar.

Reply:
```suggestion
* _Programmability_ - the extent to which the software codes operated upon inside the bounds of the TEE provided after time of manufacturer by an identified actor.
```